### DISCUSSION TITLE
mspider/user-guide/service-mesh/params

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2920] [Announcements]
# mspider/user-guide/service-mesh/params

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-mesh/params.html

**Comment by nicole-lihui** [comment]
创建参数现在新增不少，应该还可以补充一些

**Comment by samzong** [comment]
更新配图，创建网格的应用规模支持自定义更多 size 和自定义参数
